src/app/js/lib/Pagination.js

```diff
@@ -152,7 +125,7 @@ class Pagination extends Component {
 						{
 							pages.map(page => (
 								<MenuItem 
-									primaryText={page}
+									primaryText={page + 1}
```

Comment:
What if there is 10000 pages? (I had the case, it froze the browser). 
Replace with a text input.